frontend/src/pages/ActivityReport/Pages/reviewItem.js

```diff
@@ -68,6 +68,9 @@ const Item = ({ label, value, path }) => {
   if (!Array.isArray(value)) {
     values = [value];
   }
+  if (label === 'Other Resources' || label === 'Attachments') {
+    values = values.map((v) => v.originalFileName);
+  }
```

Comment:
I don't think you'd need to special case this if you utilize the `path` parameter.

Reply:
See how the review for `collaborating specialists` is handled

Reply:
you're correct. Just pushed that. much cleaner. Thanks.